examples/textConnect/config_js/config.js

```diff
@@ -55,10 +78,26 @@ jQuery.noConflict();
 
 
     function setDropdown() {
-        // フォーム設計情報を取得し、選択ボックスに代入する
+        // get the form fields info and put them in the selection boxes
         var url = kintone.api.url('/k/v1/preview/app/form/fields', true);
         kintone.api(url, 'GET', {'app': kintone.app.getId()}, function(resp) {
+
+            //
+            var template = $.templates(document.querySelector('#plugin-template'));
+            var templateItems = {
+                connectTitle: i18n.connectTitle,
+                connectDescription: i18n.connectDescription,
+                delimiterTitle: i18n.delimiterTitle,
+                delimiterDescription: i18n.delimiterDescription,
+                resultTitle: i18n.resultTitle,
+                resultDescription: i18n.resultDescription
+            };
+            $('#plugin-container').html(template(templateItems));
```

Comment:
By rewriting from line 86, the contents in line 87-94 can be shortened.

Use the Conditional Format 2 codes as reference:

html: https://github.com/kintone/plugin-examples/blob/38921de5eeec7ea1eff3f697287b8e31ce8cdd47/examples/conditionformat2/html/config.html#L7
js: https://github.com/kintone/plugin-examples/blob/38921de5eeec7ea1eff3f697287b8e31ce8cdd47/examples/conditionformat2/js/config.js#L301-L303

Reply:
You can probably remove `$('#plugin-container').html(template(templateItems));` when using this method as well

Reply:
Updated the codes. Please see the latest commit made on 8/17.